docs/source/markdown/options/secret.md

```diff
@@ -19,8 +19,20 @@ Secrets and its storage are managed using the `podman secret` command.
 
 Secret Options
 
-- `type=mount|env`    : How the secret will be exposed to the container. Default mount.
-- `target=target`     : Target of secret. Defaults to secret name.
+- `type=mount|env`    : How the secret will be exposed to the container. Defaults to `mount`.
+- `target=target`     : Target of secret.
+                        For mounted secrets, this is the path to the secret inside the container.
+                        If a fully qualified path is provided, the secret will be mounted at that location.
+                        Otherwise, the secret will be mounted to `/run/secrets/target`.
+                        If target is not set, by default the secret will be mounted to `/run/secrets/secretname`.
+                        For env secrets, this is the environment variable key. Defaults to `secretname`.
 - `uid=0`             : UID of secret. Defaults to 0. Mount secret type only.
 - `gid=0`             : GID of secret. Defaults to 0. Mount secret type only.
 - `mode=0`            : Mode of secret. Defaults to 0444. Mount secret type only.
+
+
+Examples
```

Comment:
Usually we do a `##` before this, and surround individual examples with triple-backticks

Reply:
Changed the triple backticks! I think this is under a `####` section for in a flag doc though, so I think leaving Examples as-is might be the move?